doc/modules/cross_validation.rst

```diff
@@ -196,6 +196,8 @@ following keys -
 for all the scorers. If train scores are not needed, this should be set to
 ``False`` explicitly.
 
+``return_estimator`` is set to ``False`` by default. When set to ``True``, it adds an ``estimator`` key containing the estimators fitted on each split.
```

Comment:
Please limit lines to under 80 characters

Reply:
I also think this is somewhat unnecessary. Or I'd state it more in terms of use case rather than API: "you can also retain to estimator fitted for each training set with return_estimator=True"